beta-tester/amp-beta-tester.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Plugin Name: AMP Beta Tester
+ * Description: Opt-in to receive non-stable release builds for the AMP plugin.
+ * Plugin URI: https://amp-wp.org
+ * Author: AMP Project Contributors
+ * Author URI: https://github.com/ampproject/amp-wp/graphs/contributors
+ * Version: 0.1
+ * Text Domain: amp-beta-tester
+ * Domain Path: /languages/
+ * License: GPLv2 or later
+ *
+ * @package AMP Beta Tester
+ */
+
+namespace AMP\Beta_Tester;
+
+define( 'AMP__BETA_TESTER__DIR__', dirname( __FILE__ ) );
+
+require_once AMP__BETA_TESTER__DIR__ . '/includes/class-autoloader.php';
+Autoloader::register( __NAMESPACE__ );
+
+$amp_beta_tester_plugin = new AMP_Beta_Tester_Plugin();
+add_action( 'plugins_loaded', [ $amp_beta_tester_plugin, 'init' ] );
```

Comment:
This entire file can actually be located in the root of the plugin directory. So `amp-beta-tester.php` and `amp.php` both in the same directory. When this is done, then WordPress will automatically see both of the plugins. It may make sense to put all of the code in this single file and not use any classes for the logic in it. It could all be handled with namespaced functions.

Reply:
I forgot to mention that the reason for putting the plugin in its own folder was because the WordPress Plugin Update API uses the folder of the main PHP file (`amp` in this case) as an identifier for updates. So any other plugin file that is put in the `amp` folder will have the same version number as the AMP plugin.

One workaround for that would be to utilize the `all_plugins` filter hook to correct the flaw when the plugins list table is being shown.

Reply:
Here's an example of the workaround: https://github.com/ampproject/amp-wp/pull/3560/commits/060463ec2928435d8c6b7fd86ab0d24aeaa64990#diff-4805c837d3390517773926fb26726bcdR165-R178

Reply:
The workaround doesn't work if AMP is deactivated, however:

![image](https://user-images.githubusercontent.com/16200219/67164034-c7125c00-f33b-11e9-8ee5-045898ccfc6d.png)

Reply:
I suggest including the workaround code `amp.php` as well then.